### ISSUE TITLE
Strange behaviour of loss graph

### ISSUE THREAD

**Issue by omerasif57** [harveyslash/Facial-Similarity-with-Siamese-Networks-in-Pytorch#34]
Hey, I tried to run the notebook in Google Colab and when training the loss behave this way:

![image](https://user-images.githubusercontent.com/17876048/86168165-3a0ddc80-bb31-11ea-8de2-cb21ac724a3b.png)

is it normal/expected? 
I did not tweak the code nor any parameters. Please advise.

**Comment by harveyslash** [comment]
This is normal, yes. And it is a sign of the model being able to learn.
